app/src/main/java/ru/practicum/android/diploma/presentation/search/SearchUiRenderer.kt

```diff
@@ -109,12 +165,83 @@ class SearchUiRenderer(
         binding.progressBarBottom.visibility = View.GONE
     }
 
-    private fun showMessageImage(drawableRes: Int) {
+    /**
+     * Показать изображение с сообщением
+     */
+    fun showMessageImage(drawableRes: Int) {
         binding.messageImage.apply {
             setImageDrawable(
                 ContextCompat.getDrawable(binding.root.context, drawableRes)
             )
             visibility = View.VISIBLE
         }
     }
+
+    /**
+     * Обновить прогресс-бар нижней загрузки
+     */
+    fun updateBottomProgressBar(isVisible: Boolean) {
+        binding.progressBarBottom.visibility = if (isVisible) View.VISIBLE else View.GONE
```

Comment:
Можно использовать экстеншн isVisible:

```suggestion
        binding.progressBarBottom.isVisible = iisVisible
```